contrib/test/ci/vars.yml

```diff
@@ -130,6 +130,10 @@ kata_skip_ctr_tests:
   - 'test "ctr with default_env set in configuration"'
   - 'test "ctr with absent mount that should be rejected"'
   - 'test "ctr stop loop kill retry attempts"'
+  - 'test "ctr that mounts container storage as read-only option but not recursively"'
+  - 'test "ctr that mounts container storage as recursively read-only"'
+  - 'test "ctr that fails to mount container storage as recursively read-only without readonly option"'
+  - 'test "ctr that fails to mount container storage as recursively read-only without private propagation"'
```

Comment:
**To the reviewer:**

I am not sure if Kata supports this correctly. I couldn't get it to work. Perhaps it's the [virtio-fs](https://virtio-fs.gitlab.io/) that's behind the scenes that requires some setting to be correctly enabled.

Reply:
I can't answer from the top of my head. It's probably better to keep them in the skip list for now.
Thanks!